src/test/java/apicurito/tests/steps/ConfigurationOCPSteps.java

```diff
@@ -197,12 +209,14 @@ public void deleteRunningInstanceOfApicurito() {
 
     @Then("clean openshift after operatorhub test")
     public void cleanOpenshiftAfterOperatorhubTest() {
-        String csv =
-            OpenShiftUtils.getInstance().apps().deployments().inNamespace(TestConfiguration.openShiftNamespace()).withName("fuse-apicurito").get()
-                .getMetadata().getOwnerReferences().get(0).getName();
-        OpenShiftUtils.binary().execute("delete", "csv", csv);
+//        String csv = OpenShiftUtils.getInstance().apps().deployments()
+//            .inNamespace(TestConfiguration.openShiftNamespace()).withName("fuse-apicurito").get().getMetadata()
+//            .getOwnerReferences().get(0).getName();
+        OpenShiftUtils.binary().execute("delete", "csv", "fuse-apicurito.v7.9.0");
         OpenShiftUtils.binary().execute("delete", "subscription", "fuse-apicurito");
-        OpenShiftUtils.binary().execute("delete", "CatalogSource", "apicurito-test-catalog", "-n", "openshift-marketplace");
+//        OpenShiftUtils.binary().execute("delete", "operatorgroup", "fo-operatorgroup");
+//        OpenShiftUtils.binary().execute("delete", "CatalogSource", "apicurito-test-catalog", "-n",
+//            "openshift-marketplace");
```

Comment:
These lines can be deleted or executed `if (TestConfiguration.namespaceCleanupAfter())`